server/kv.go

```diff
@@ -16,14 +16,15 @@ import (
 )
 
 const (
-	keyInstances        = "instances/v3"
-	keyRSAKey           = "rsa_key"
-	keyTokenSecret      = "token_secret"
-	prefixInstance      = "conf_instance_"
-	prefixUser          = "user_"
-	prefixOneTimeSecret = "ots_" // + unique key that will be deleted after the first verification
-	webhookKeyPrefix    = "webhook"
-	configKeyPrefix     = "_config"
+	keyInstances              = "instances/v3"
+	keyRSAKey                 = "rsa_key"
+	keyTokenSecret            = "token_secret"
+	prefixInstance            = "conf_instance_"
+	prefixUser                = "user_"
+	prefixOneTimeSecret       = "ots_" // + unique key that will be deleted after the first verification
+	webhookKeyPrefix          = "webhook"
+	configKeyPrefix           = "_config"
```

Comment:
shouldn't it be like this `prefixWebhookKey` and `prefixConfigKey`?

Reply:
they have followed these naming conventions everywhere that's why I have followed this

Reply:
fixed